Reword covered use case language

smishra1200 opened the pull request:
suggestion to remove "this solution may address uses cases" -> this solution shall address use cases that may include...

smishra1200 committed:
Update charter.md

suggestion to remove "this solution may address uses cases" -> this solution shall address use cases that may include...

fluffy commented:
What would you think of changing "shall address" to "addresses" ? I don't feel strongly about this one way or other

smishra1200 commented:
Hi Cullen, changing "shall address" to "addresses" works for me

smishra1200 committed:
Update charter.md

changed "shall address" -> addresses

suhasHere commented:
+1 on the change

smishra1200 commented:
I updated the text as Cullen suggested, ie, changed from "shall address" to
"addresses".

-Sanjay

On Thu, Jul 28, 2022 at 4:15 PM Suhas Nandakumar ***@***.***>
wrote:

> +1 on the change
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_moq-2Dwg_moq-2Dcharter_pull_63-23issuecomment-2D1198588914&d=DwMCaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=TEDGyOmr72xmjdt3Xm3JM8JGm2yQuOJKCXFpmt1of-HTQqRRRVl3fBJujKGzFBHM&s=Ylw9LxO3Xu3c1AMuhdRaiTue3-OaBdmiRHF_CXMhTvg&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABGNJTEFANGH5ZML5KMYRHDVWLS6BANCNFSM542CA32Q&d=DwMCaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=TEDGyOmr72xmjdt3Xm3JM8JGm2yQuOJKCXFpmt1of-HTQqRRRVl3fBJujKGzFBHM&s=8_r1juzUK0xLtlDp4yyOM5wz38JjcmsDTvJUZDXMSl0&e=>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

hardie commented:
Hi Sanjay,

The MoQ charter was already approved by the IESG, so changes at this level
don't go through this process any more.

regards,

Ted

On Tue, Oct 11, 2022 at 9:50 AM Sanjay Mishra ***@***.***>
wrote:

> I updated the text as Cullen suggested, ie, changed from "shall address" to
> "addresses".
>
> -Sanjay
>
> On Thu, Jul 28, 2022 at 4:15 PM Suhas Nandakumar ***@***.***>
> wrote:
>
> > +1 on the change
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_moq-2Dwg_moq-2Dcharter_pull_63-23issuecomment-2D1198588914&d=DwMCaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=TEDGyOmr72xmjdt3Xm3JM8JGm2yQuOJKCXFpmt1of-HTQqRRRVl3fBJujKGzFBHM&s=Ylw9LxO3Xu3c1AMuhdRaiTue3-OaBdmiRHF_CXMhTvg&e=
> >,
> > or unsubscribe
> > <
> https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABGNJTEFANGH5ZML5KMYRHDVWLS6BANCNFSM542CA32Q&d=DwMCaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=TEDGyOmr72xmjdt3Xm3JM8JGm2yQuOJKCXFpmt1of-HTQqRRRVl3fBJujKGzFBHM&s=8_r1juzUK0xLtlDp4yyOM5wz38JjcmsDTvJUZDXMSl0&e=
> >
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/moq-wg/moq-charter/pull/63#issuecomment-1274335596>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKVXZEN6Z44UWXPHVHWMVDWCUS47ANCNFSM542CA32Q>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

smishra1200 commented:
Hi Ted - I am good with the final MOQ charter as the language is clear WRT to the statement about the use cases. The PR I raised right after the IETF 114 meeting was specifically asking to change 
from: "this solution may address uses cases" 
to: "this solution shall address use cases that may include..."

In the end, the final wording is even better: "This solution addresses use cases including live streaming, gaming, and media conferencing and will scale efficiently."

So, I am all good WRT my PR, however, administratively, it appears my request should have been picked as it was submitted way ahead of the IESG request and in fact, I spoke about this on the MIC in the room (the day of the BOF).

smishra1200 commented:
Hi Ted - I am good with the final MOQ charter as the language is clear WRT
to the statement about the use cases. The PR
<https://github.com/moq-wg/moq-charter/pull/63> I raised right after the
IETF 114 meeting was specifically asking to change:

from: "this solution may address uses cases"
to: "this solution shall address use cases that may include..."

In the end, the final wording is even better: "This solution addresses use
cases including live streaming, gaming, and media conferencing and will
scale efficiently."

So, I am all good WRT my PR, however, administratively, it appears my
request should have been picked as it was submitted way ahead of the IESG
request and in fact, I spoke about this on the MIC in the room (the day of
the BOF).

Thanks
Sanjay

On Tue, Oct 11, 2022 at 5:54 AM hardie ***@***.***> wrote:

> Hi Sanjay,
>
> The MoQ charter was already approved by the IESG, so changes at this level
> don't go through this process any more.
>
> regards,
>
> Ted
>
> On Tue, Oct 11, 2022 at 9:50 AM Sanjay Mishra ***@***.***>
> wrote:
>
> > I updated the text as Cullen suggested, ie, changed from "shall address"
> to
> > "addresses".
> >
> > -Sanjay
> >
> > On Thu, Jul 28, 2022 at 4:15 PM Suhas Nandakumar ***@***.***>
> > wrote:
> >
> > > +1 on the change
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_moq-2Dwg_moq-2Dcharter_pull_63-23issuecomment-2D1198588914&d=DwMCaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=TEDGyOmr72xmjdt3Xm3JM8JGm2yQuOJKCXFpmt1of-HTQqRRRVl3fBJujKGzFBHM&s=Ylw9LxO3Xu3c1AMuhdRaiTue3-OaBdmiRHF_CXMhTvg&e=
> > >,
> > > or unsubscribe
> > > <
> >
> https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABGNJTEFANGH5ZML5KMYRHDVWLS6BANCNFSM542CA32Q&d=DwMCaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=TEDGyOmr72xmjdt3Xm3JM8JGm2yQuOJKCXFpmt1of-HTQqRRRVl3fBJujKGzFBHM&s=8_r1juzUK0xLtlDp4yyOM5wz38JjcmsDTvJUZDXMSl0&e=
> > >
> > > .
> > > You are receiving this because you authored the thread.Message ID:
> > > ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/moq-wg/moq-charter/pull/63#issuecomment-1274335596>
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_moq-2Dwg_moq-2Dcharter_pull_63-23issuecomment-2D1274335596-253E&d=DwQFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=CuuztuCMZ5Cjz_cJ3VEKb0Usv8RwSlhKQvPNVg49h6WYUd9lj8TrUfDaAI03IHK5&s=8mvGddPBs20QosAXTF9eV8AEhQN3L04rZ0NXJMyknaU&e=>
> ,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAKVXZEN6Z44UWXPHVHWMVDWCUS47ANCNFSM542CA32Q>
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AAKVXZEN6Z44UWXPHVHWMVDWCUS47ANCNFSM542CA32Q-253E&d=DwQFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=CuuztuCMZ5Cjz_cJ3VEKb0Usv8RwSlhKQvPNVg49h6WYUd9lj8TrUfDaAI03IHK5&s=KJTGID7zGzBIQUUR3KRmNvUpdSyfQn7b93kViqzpliI&e=>
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_moq-2Dwg_moq-2Dcharter_pull_63-23issuecomment-2D1274432627&d=DwMFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=CuuztuCMZ5Cjz_cJ3VEKb0Usv8RwSlhKQvPNVg49h6WYUd9lj8TrUfDaAI03IHK5&s=pSdCcPLljSaKkVuWd7ZaaY3UT6AuP6vYgVzDKhi3PL8&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABGNJTBGL7RLQQSMIA7NKIDWCU2L3ANCNFSM542CA32Q&d=DwMFaQ&c=udBTRvFvXC5Dhqg7UHpJlPps3mZ3LRxpb6__0PomBTQ&r=XniVbishGiO2Ao9hKqSc-hTVIWCi3T-x6GdHR4ZTgoM&m=CuuztuCMZ5Cjz_cJ3VEKb0Usv8RwSlhKQvPNVg49h6WYUd9lj8TrUfDaAI03IHK5&s=4n5PST4XqamQ-GwGL5SGgeCL9kAqnAPUa35UfYmGPDk&e=>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
